DataConnectors/Forcepoint CASB.json

```diff
@@ -0,0 +1,111 @@
+{
+    "id": "Forcepoint",
+    "title": "Forcepoint CASB",
+    "publisher": "Forcepoint CASB",
+    "descriptionMarkdown": "The Forcepoint CASB (Cloud Access Security Broker) Connector allows you to automatically export CASB logs and events into Azure Sentinel in real-time. This enriches visibility into user activities across locations and cloud applications, enables further correlation with data from Azure workloads and other feeds, and improves monitoring capability with Workbooks inside Azure Sentinel​",
```

Comment:
I see weird chars after end of the sentence (description). Pasting as follows. Please fix by deleting "Sentinel." and click delete button after the '.' and retype 'Sentinel.'
'improves monitoring capability with Workbooks inside Azure Sentinelâ'